xml/ssh_banners.xml

```diff
@@ -888,9 +888,10 @@
     <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:13.04"/>
   </fingerprint>
 
-  <fingerprint pattern="^OpenSSH_(6\.2p2) (Ubuntu-6unbuntu\d(?:\.\d)?)$">
+  <fingerprint pattern="^OpenSSH_(6\.2p2) (Ubuntu-6\S*)$">
     <description>OpenSSH running on Ubuntu 13.10</description>
     <example service.version="6.2p2" openssh.comment="Ubuntu-6unbuntu0.4">OpenSSH_6.2p2 Ubuntu-6unbuntu0.4</example>
```

Comment:
Interesting, the original pattern and example look like they contain a typo  uNbuntu `6unbuntu`.

Reply:
I think it's a typo, and it should be fixed.
@hdm can you confirm?

Reply:
This was intentional, the older pattern didn't match all variants, this one does via \S+